charts/gardener-extension-networking-calico/templates/clusterfilters-logging.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: fluentbit.fluent.io/v1alpha2
+kind: ClusterFilter
+metadata:
+  labels:
+    fluentbit.gardener/type: "seed"
+  name: calico-node-calico-node
+spec:
+  filters:
+  - parser:
+      keyName: log
+      parser: caliconode-parser
+      reserveData: true
+  match: kubernetes.*calico-node*calico-node*
```

Comment:
To match only log streams that are coming from podName `calico-node` and containerName `calico-node`.
Same as it is now: https://github.com/gardener/gardener-extension-networking-calico/blob/master/charts/gardener-extension-networking-calico/templates/configmap-logging.yaml#L12

Reply:
Ok, I suppose we used the same match clause before.
Just out of curiosity: Why is it `kubernetes.*calico-node*calico-node*` and not `kubernetes.*calico-node.*calico-node.*` or kubernetes*calico-node*calico-node*? Or to put it in different terms: Could you please provide an example that this pattern is matched against?
It seems we use it already. Hence, it should hopefully work. Nevertheless, it would help to understand what we match here at all.

Reply:
all of the logs which are coming from the tail plugin are prefixed with tag kubernetes.: https://github.com/gardener/gardener/blob/master/charts/seed-bootstrap/charts/fluent-bit/templates/_fluent-bit-config.tpl#L23

> Why is it kubernetes.*calico-node*calico-node* and not kubernetes.*calico-node.*calico-node.*

I do not remember well but I think that it was not a generic regular expression.
So this means that it will match every tag which starts with `kubernetes.` then will have `calico-node` and then again `calico-node`

As I remember the tag was containing also the docker_id of the container